test/cypress/integration/worldpay-3ds-flex-ddc/payment-with-session-id.cy.test.js

```diff
@@ -175,9 +120,47 @@ describe('Worldpay 3ds flex card payment flow', () => {
     })
   })
 
-  setUpAndCheckCardPaymentPage()
   describe('Worldpay responds with status=false', () => {
-    it('submitting confirmation should not include the hidden input containing the session ID', () => {
+    it('Should not include the hidden input containing the session ID when submitting confirmation ', () => {
+      cy.task('setupStubs', createPaymentChargeStubsEnglish)
+      cy.visit(`/secure/${tokenId}`)
+
+      // 1. Charge will be created using this id as a token (GET)
+      // 2. Token will be deleted (DELETE)
+      // 3. Charge will be fetched (GET)
+      // 4. Service related to charge will be fetched (GET)
+      // 5. Charge status will be updated (PUT)
+      // 6. Client will be redirected to /card_details/:chargeId (304)
+      cy.location('pathname').should('eq', `/card_details/${chargeId}`)
+
+      cy.log('Should enter and validate a correct card')
+
+      cy.task('clearStubs')
+      cy.task('setupStubs', checkCardDetailsStubs)
+
+      cy.server()
+      cy.route('POST', `/check_card/${chargeId}`).as('checkCard')
+
+      cy.log('Should enter payment details')
```

Comment:
Can the code to enter payment details be extracted into a separate method for reuse?

Reply:
Good idea - will do that.

Reply:
Is this done?